doc/lua_api.txt

```diff
@@ -971,33 +971,92 @@ There are a bunch of different looking node types.
 Look for examples in `games/minimal` or `games/minetest_game`.
 
 * `normal`
+    * A node-sized cube.
 * `airlike`
+    * Invisible, uses no texture.
 * `liquid`
+    * The cubic source node for a liquid.
 * `flowingliquid`
+    * The flowing version of a liquid, appears with various heights and slopes.
 * `glasslike`
+    * Often used for partially-transparent nodes.
+    * Only external sides of textures are visible.
 * `glasslike_framed`
+    * All face-connected nodes are drawn as one volume within a surrounding
+      frame.
+    * The frame appearence is generated from the edges of the first texture
+      specified in `tiles`. The width of the edges used are 1/16th of texture
+      size: 1 pixel for 16x16, 2 pixels for 32x32 etc.
+    * The glass 'shine' (or other desired detail) on each node face is supplied
+      by the second texture specified in `tiles`.
 * `glasslike_framed_optional`
-* `allfaces`
+    * This switches between the above 2 drawtypes according to the menu setting
+      'Connected Glass'.
```

Comment:
The menu setting string depends on the language. Anyway, it's bikeshedding.

Reply:
lua_api.txt is in English :)